Makefile

```diff
@@ -0,0 +1,11 @@
+setup:
+	poetry2setup > setup.py
+
+build: setup shiv
+
+clean:
+	rm setup.py
+
+shiv:
+	mkdir -p build
+	shiv -c bubbles -o build/bubbles.pyz . --compressed

```

Comment:
Not sure how I feel about having a `Makefile`, but I guess if it works...

Reply:
Makefiles are wonderful 😂 Tbh I'm not a huge fan of them either but as you say, they work and it's relatively easy to extend. If you have any better ideas, I'm open to suggestions!